controllers/user.js

```diff
@@ -7,10 +7,44 @@ const createUser = async (req, res) => {
     await user.save();
     res.status(201).json(user);
   } catch (err) {
-    res.status(400).send({status: 400, message: err.message});
+    res.status(400).send({ status: 400, message: err.message });
+  }
+};
+
+const fetchUserByID = async (req, res) => {
+  const _id = req.params.id;
```

Comment:
Not sure why the underscore, but it looks good 😆

Reply:
yeah, the naming is a bit odd. Fixed!